inference-engine/include/vpu/vpu_config.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief A header that defines common config subset for VPU plugins.
+ * Include myriad_config.hpp or hddl_config.hpp directly.
+ * These properties should be used in SetConfig() and LoadNetwork() methods of plugins
+ *
+ * @file vpu_config.hpp
+ */
+
+#pragma once
+
+#include "ie_plugin_config.hpp"
+#include "ie_api.h"
+
+#include <string>
+
+#define DECLARE_VPU_CONFIG(name) static constexpr auto name = #name
+
+namespace InferenceEngine {
+
+//
+// Common options
+//
+
+/**
+ * @brief Turn on HW stages usage (applicable for MyriadX devices only).
+ * This option should be used with values: CONFIG_VALUE(YES) (default) or CONFIG_VALUE(NO)
```

Comment:
Nitpick: I'd move mentioning default value on the separate line.

Example:

```
/**
 * @brief Turn on HW stages usage (applicable for MyriadX devices only).
 * The only possible values are: 
 *     CONFIG_VALUE(YES)
 *     CONFIG_VALUE(NO)
 * Default value is CONFIG_VALUE(YES)
 */
```

or 

```
/**
 * @brief Turn on HW stages usage (applicable for MyriadX devices only).
 * The only possible values are: 
 *     CONFIG_VALUE(YES) (default value)
 *     CONFIG_VALUE(NO)
 */
```